dd-sdk-android/src/test/kotlin/com/datadog/android/DatadogConfigBuilderTest.kt

```diff
@@ -581,4 +583,89 @@ class DatadogConfigBuilderTest {
                 )
             )
     }
+
+    @Test
+    fun `builder with all tracking instrumentation disabled`(forge: Forge) {
+        val rumUrl = forge.aStringMatching("http://[a-z]+\\.com")
+        val config = DatadogConfig.Builder(fakeClientToken, fakeApplicationId)
+            .useCustomRumEndpoint(rumUrl)
+            .build()
+
+        assertThat(config.rumConfig)
+            .isEqualTo(
+                DatadogConfig.RumConfig(
+                    fakeClientToken,
+                    fakeApplicationId,
+                    rumUrl,
+                    DatadogConfig.DEFAULT_SERVICE_NAME,
+                    DatadogConfig.DEFAULT_ENV_NAME,
+                    trackGestures = false,
+                    trackActivitiesAsScreens = false,
+                    trackFragmentsAsScreens = false
+                )
+            )
+    }
+
+    @Test
+    fun `builder with track gestures enabled`(forge: Forge) {
+        val rumUrl = forge.aStringMatching("http://[a-z]+\\.com")
+        val config = DatadogConfig.Builder(fakeClientToken, fakeApplicationId)
+            .useCustomRumEndpoint(rumUrl)
+            .trackGestures()
+            .build()
+
+        assertThat(config.rumConfig)
+            .isEqualTo(
+                DatadogConfig.RumConfig(
+                    fakeClientToken,
+                    fakeApplicationId,
+                    rumUrl,
+                    DatadogConfig.DEFAULT_SERVICE_NAME,
+                    DatadogConfig.DEFAULT_ENV_NAME,
+                    trackGestures = true
+                )
+            )
+    }
+
+    @Test
+    fun `builder with track activities as screens enabled`(forge: Forge) {
+        val rumUrl = forge.aStringMatching("http://[a-z]+\\.com")
+        val config = DatadogConfig.Builder(fakeClientToken, fakeApplicationId)
+            .useCustomRumEndpoint(rumUrl)
+            .trackActivitiesAsScreens()
+            .build()
+
+        assertThat(config.rumConfig)
+            .isEqualTo(
+                DatadogConfig.RumConfig(
+                    fakeClientToken,
+                    fakeApplicationId,
+                    rumUrl,
+                    DatadogConfig.DEFAULT_SERVICE_NAME,
+                    DatadogConfig.DEFAULT_ENV_NAME,
+                    trackActivitiesAsScreens = true
```

Comment:
Do we want to allow tracking both screens and activities as view ? The way the RUM monitor works, only one view can be active at any time, meaning that you'll start a view for an activity, and immediately after start a view for a fragment.

Reply:
That's a good point here, I don't think we should allow both in the same time. I will change in the way that if you selected "trackFragmentsAsScreens" to override the "trackActivityAsScreens" flag to false (meaning that fragments screens have priority). I will state this in the method docs also.